flocker/node/agents/test/test_blockdevice.py

```diff
@@ -169,12 +169,10 @@
 
 DESIRED_DATASET_ATTRIBUTE_STRATEGIES = {
     'dataset_id': uuids(),
-    'maximum_size': integers(min_value=0).map(
-        lambda n: (
-            LOOPBACK_MINIMUM_ALLOCATABLE_SIZE +
-            n * LOOPBACK_ALLOCATION_UNIT
-        )
-    ),
+    'maximum_size': integers(
```

Comment:
I agree.

Our general philosophy with hypothesis in CHQ seems to be "assume you are forgetting about some edge case, be very liberal with your strategies, and let hypothesis show you".

I think I agree with that strategy _if_ the tests you are running are actually unit tests (can run in under 1ms). In this particular case, loop devices take a lot longer to interact with than 1ms, so I think we should trim the search space for hypothesis. Thus, I think you are correct, we should use a `just(LOOPBACK_MINIMUM_ALLOCATABLE_SIZE)` for this. Especially considering loop devices are only intended for use in test code. exercising weird corner cases doesn't get us much.